test/ConcreteOperations/convex_hull.jl

```diff
@@ -185,4 +185,9 @@ for N in [Float64, Rational{Int}]
     # binary convex hull with an empty set is identical to unary
     @test convex_hull(S, EmptySet{N}(1)) == convex_hull(EmptySet{N}(1), S) == S
     @test convex_hull(P, EmptySet{N}(2)) == convex_hull(EmptySet{N}(2), P) == Pc
+
+    # see #3455
+    v = [N[-0.0, 2], N[0, 0], N[1, 0], N[0, 1], N[0, 3//2]]
```

Comment:
Sorry, I accidentally modified the example in my previous suggestion. This change will fix it.

```suggestion
    v = [N[-0.0, 2], N[2, 0], N[1, 0], N[0, 1], N[0, 3//2]]
```